internal/pkg/ntp/ntp.go

```diff
@@ -126,12 +140,75 @@ func (syncer *Syncer) restartSync() {
 	}
 }
 
+func absDuration(d time.Duration) time.Duration {
```

Comment:
Another place where generics would be useful 😆

Reply:
yep, exactly :)